frontend/src/stores/serviceStore.ts

```diff
@@ -84,12 +85,18 @@ export class ServiceStore extends DomainStore<Service, ServiceCollection> {
     const response = await this.mainStore.api.put<ServiceCollection[]>('/services/' + service.id, service);
     this.services = response.data;
   }
+
+  private async callServiceCalculator<ReturnType>(url: string) {
+    const response = await this.mainStore.api.
+    get<ReturnType>(url);
```

Comment:
Why is this on a new line?
Anyway, I would preder this method in serviceStore as static method and URLs as helpers